src/main/java/sirius/kernel/commons/URLBuilder.java

```diff
@@ -70,6 +74,55 @@ public URLBuilder(@Nonnull String protocol, @Nonnull String host) {
         url.append(host);
     }
 
+    /**
+     * Adds a path part to the url.
```

Comment:
Sollten wir das als Abkürzung nicht groß schreiben?

```suggestion
     * Adds a path part to the URL.
```